server/keymanapp-test-bot/keymanapp-test-bot.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Keyman is copyright (C) SIL International. MIT License.
+ *
+ * @keymanapp-test-bot implementation
+ */
+
+import { ManualTestProtocol, ManualTestStatus, ManualTestUtil } from "../../shared/manual-test/manual-test-protocols";
+import ManualTestParser from '../../shared/manual-test/manual-test-parser';
+import { Octokit } from "@octokit/core";
+import { restEndpointMethods } from "@octokit/plugin-rest-endpoint-methods";
+import { User } from "@octokit/webhooks-types";
+import { Probot, ProbotOctokit } from "probot";
+
+const manualTestRequiredLabelName = 'user-test-required';
+const manualTestMissingLabelName = 'user-test-missing';
+
+function log(s) {
+  console.log('@keymanapp-test-bot: '+s);
+}
+
+interface ProcessEventData {
+  owner: string;
+  repo: string;
+  issue_number: number;
+};
+
+async function processEvent(
+  octokit: InstanceType<typeof ProbotOctokit>,
+  data: ProcessEventData,
+  is_pull_request: boolean
+) {
+
+  // TODO: we may just be able to use the issue data coming in
+  const issue = await octokit.rest.issues.get({...data});
```

Comment:
So are issue/pr the same as far as this parser is concerned. differentiated by certain defines

Reply:
Yep, pull requests are essentially issue+code in GitHub, which is very convenient!